### PR TITLE
:white_check_mark: Tested with latest version and updated the docs

### PR DESCRIPTION
**Pull request by LuffySama-Dev**
Hi @barriebyron ,

Good Day!! 

I have tested this example for issue #755 .
I found the docs where still not updated to I have updated them and the Readme as well.

Can you please help to review this PR and let me know if any changes are required ?

Thanks 😃

### CODE REVIEW

**Comment by barriebyron** [comment]
thank you @LuffySama-Dev  looks good, please accept the suggestions and tag me again and we'll get this PR merged

**Comment by LuffySama-Dev** [comment]
Hi @barriebyron ,

I am unable to see any comments. 

Thank You 😃

**Review by barriebyron — COMMENTED** [review]
@LuffySama-Dev  thanks for being on top of this, do you see the suggestions now?

File: examples/zkapps/05-common-types-and-functions/README.md [hunk 1]
```diff
@@ -1,25 +1,53 @@
...
+    ```sh
+    npm run build && node build/src/main.js
+    ```
+
+## Expected output:
+![05output](05output.png)
+
+## Last Audit Date:
+ 
+**22-January-2024**
```
[the remark this answers is not in the document]
Comment (LuffySama-Dev) [reply examples/zkapps/05-common-types-and-functions/README.md]: Thanks 😃

**Comment by LuffySama-Dev** [comment]
Hi @barriebyron ,

Changes are applied, can you please help to review ?

Thanks 😃

**Review by barriebyron — APPROVED** [review]
thanks @LuffySama-Dev

**Pull request merged**